shell/platform/android/AndroidManifest.xml

```diff
@@ -3,18 +3,18 @@
      Use of this source code is governed by a BSD-style license that can be
      found in the LICENSE file.
  -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="org.domokit.sky.shell" android:versionCode="1" android:versionName="0.0.1">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.flutter.app" android:versionCode="1" android:versionName="0.0.1">
```

Comment:
I think the place that needs the comment is https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/create/android/AndroidManifest.xml.tmpl

I think this file is different in that it kind of wants to be in the io.flutter.app package and isn't meant to be user editable.